lib/collection/src/operations/types.rs

```diff
@@ -129,6 +129,8 @@ pub struct CollectionInfo {
     pub config: CollectionConfig,
     /// Types of stored payload
     pub payload_schema: HashMap<PayloadKeyType, PayloadIndexInfo>,
+    /// Properties or metadata of the collection
+    pub comment: String,
```

Comment:
To make it backward compatible, let's make this field optional and skip serialization if none